daffodil-core/src/main/scala/org/apache/daffodil/unparsers/runtime1/PackedBinaryUnparserTraits.scala

```diff
@@ -100,6 +102,14 @@ abstract class PackedBinaryDecimalBaseUnparser(
         binaryDecimalVirtualPoint
       )
     }
+    if (bigDec.signum() == -1 && decimalSigned == YesNo.No) {
+      UnparseError(
+        One(e.schemaFileLocation),
+        Nope,
+        "Packed binary decimal value is negative, but decimalSigned is no",
+        context.dpathElementCompileInfo.namedQName.toPrettyString
```

Comment:
The format string doesn't have a `%s` in it, so I don't think this argument is going to be used. I think that's fine, since I *think* the namedQName should already be in the diagnostic context anyways? So I would suggest just removing the argument.